src/EnergyPlus/ZoneEquipmentManager.hh

```diff
@@ -122,9 +122,20 @@ namespace ZoneEquipmentManager {
                              Real64 &FinalTotalReturnMassFlow // Final total return air mass flow rate
     );
 
+    void CalcZoneInfiltrationFlows(EnergyPlusData &state,
+                                   int const ZoneNum,                // current zone index
+                                   Real64 &ZoneReturnAirMassFlowRate // zone total zone return air mass flow rate
+    );
+
+    void ZoneReturnFlowsMaximum(EnergyPlusData &state,
+        int const ZoneNum,
+        Real64 &MaximumZoneReturnMassFlow // maximum zone total return air mass flow rate
+    );
+
```

Comment:
The zone total return flow rate is now capped at the air loop design supply air flow rate and checked for each zone air flow balancing method. For this reason wrapped the calculation in a new function.